exchange/auction.go

```diff
@@ -137,6 +144,27 @@ func maybeMake(shouldMake bool, capacity int) []prebid_cache_client.Cacheable {
 	return nil
 }
 
+func cacheTTL(impTTL int64, bidTTL int64, buffer int64) (ttl int64) {
```

Comment:
Just wanted to clarify I'm understanding the logic correctly:

1. If both imp and bid TTL are <= 0, ttl is 0
2. If just one of imp or bid TTL is <= 0, use the one that's not (+ buffer)
3. If both imp and bid TTL are > 0, use the lesser of the two (+ buffer)

Reply:
One way you can make this more readable is to "return" as soon as you know what the return value is.

```
if impTTL <= 0 {
  return bidTTL
}
if bidTTL <= 0 {
  return impTTL
}
if impTTL < bidTTL {
  return impTTL
}
return bidTTL
```

Advantage: others don't have to read through _all_ the code to decipher the behavior.
Disadvantage: it's tough do add the buffer logic properly.

You could work around that by splitting `addBuffer(base int64, buffer in64) int64` into its own function, though.

I'm ok with it as is, too, but... food for thought.